mozmeao-fr/bedrock-prod/canary-deploy.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: bedrock-prod-web-canary
   namespace: bedrock-prod
 spec:
+  progressDeadlineSeconds: 900
```

Comment:
What is this needed for?

Reply:
it just prevents the retry of creating the container from happening forever - gives a total number of seconds to try a deploy before failing.   If it's not building new or deleting old pods for the 900 seconds k8s will just mark the deployment as failed. Makes our pipelines actually fail instead of running forever.

Reply:
👍